app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace App\Http\Controllers;
```

Comment:
this is ok - but I think it will be better (And matches our architecture) to put the requests as request classes. please. but it looks good :)